src/bin/Makefile

```diff
@@ -7,7 +7,7 @@ BINFILES = align-equal align-equal-compiled acc-tree-stats \
         show-alignments compile-questions cluster-phones \
         compute-wer compute-wer-bootci make-h-transducer \
```

Comment:
Thanks, and also, IIRC, something needs to be done to `lm/kaldi-lm.a` due to removed source files there, but I do not remember the incantation. Do you?